frontend/src/pages/SignIn.tsx

```diff
@@ -164,29 +162,26 @@ export class SignIn extends React.Component<Props, State> {
 
   renderNonProcessing() {
     return (
-      <div>
-        {this.state.message && (
-          <div className={"alert " + this.state.messageClass} role="alert">
-            {this.state.message}
-          </div>
-        )}
-        <br />
-        <GoogleLogin
-          clientId={process.env.REACT_APP_GOOGLE_SIGNIN_CLIENT_ID || ""}
-          buttonText="Login"
-          onSuccess={this.responseGoogleSuccess.bind(this)}
-          onFailure={this.responseGoogleFailure.bind(this)}
-          cookiePolicy="single_host_origin"
-          prompt="select_account"
-        />
-        <br />
-        <br />
-        <GoogleLogout
```

Comment:
You're removing the only way to log out... not that it's easy to find (you have to type in "sign-in" in the URL to get there). We should probably add a "log out" button in the top right nav. 🤔 

Let's do that in a separate PR.